llvm/unittests/CAS/CASTestConfig.cpp

```diff
@@ -15,7 +15,8 @@ using namespace llvm::cas;
 
 CASTestingEnv createInMemory(int I) {
   std::unique_ptr<ObjectStore> CAS = createInMemoryCAS();
-  return CASTestingEnv{std::move(CAS)};
+  std::unique_ptr<ActionCache> Cache = createInMemoryActionCache();
+  return CASTestingEnv{std::move(CAS), std::move(Cache)};
```

Comment:
```suggestion
  return CASTestingEnv{createInMemoryCAS(), createInMemoryActionCache()};
```
Seems simpler, IMO, but totally up to you.